### PR TITLE
update firewalld formula for firewalld > 0.6

### PR DESCRIPTION
**Pull request by aboe76**
Add firewalld config with updated information from 0.6.x version
Add backend option to install backend with 0.6.x you can choose your backend, nftables or iptables
defaults to nftables

### CODE REVIEW

File: pillar.example [hunk 1]
```diff
@@ -2,6 +2,7 @@
 firewalld:
   enabled: True
   ipset: True
+  installbackend: False
```
Comment (javierbertoli) [inline pillar.example]: @aboe76, how about adding a hash like:

```yaml
firewalld:
  backend:
    manage: true
    pkg: nftables
```

It looks cleaner to me. (probably we can do the same with `ipset`?

Comment (aboe76) [reply pillar.example]: @javierbertoli yes you are right, can we change it after this merge?

Comment (javierbertoli) [reply pillar.example]: Yeah, makes sense.

**Comment by aboe76** [comment]
@javierbertoli thanks for this

**Pull request merged**